app/views/record/_sidebar.html.erb

```diff
@@ -8,11 +8,5 @@
     </div>
   <% end %>
 
-  <div class="bit askus">
```

Comment:
Aren't we losing this default contact logic the way this is implemented?

I wonder if we could continue down the path you started here and have a single `shared/ask` partial and within that partial split out the gdt/non-gdt logic?

Reply:
Yeah, I think that makes sense. The original 'Ask Us' bit felt kind of placeholder-y, but that's not my assumption to make.